workloads/templates/vm-creation.md

```diff
@@ -0,0 +1,106 @@
+# Virtual Machine Creation
+
+## Overview
+
+KubeVirt adds new types to the Kubernetes API to manage Virtual Machines and, in OpenShift, you can interact with these new resources via `oc` command as you would with any other Kubernetes API resource.
+
+In order to create a virtual machine via OpenShift CLI, you need to provide a template defining the corresponding object and its metadata.
+
+!> Only `OfflineVirtualMachine` object is currently supported.
+
+
+## Example template
+
+Here is an example template that defines an instance of the `OfflineVirtualMachine` object:
+
+\([cluster/vm-template-fedora.yaml](cluster/vm-template-fedora.yaml)\):
+
+```yaml
```

Comment:
For readability, shorten this example to the bare minimal informations required to illustrate the teplate (it does not need to work by copy and pasting). Just keep the link above to a working example.

Things to keep:
- fields where the parameters are used
- one or two fileds where NO parameters are used (to show that we can have non-parametrized fields)